dartsim/src/SimulationFeatures.cc

```diff
@@ -216,9 +216,14 @@ dart::constraint::ContactSurfaceParams GzContactSurfaceHandler::createParams(
   typedef FeaturePolicy3d P;
   typename F::ContactSurfaceParams<P> pGz;
 
+#if DART_VERSION_AT_LEAST(6, 13, 0)
```

Comment:
Not critical as part of this pull request, but I think I would prefer to see a feature macro defined somewhere like:

```
#if DART_VERSION_AT_LEAST(6,13,0)
#define DART_HAS_FRICTION_FEATURE
#endif
```
Which would make it a bit more obvious what this code is needing.